README.md

```diff
@@ -63,6 +63,7 @@ If you are unfamiliar with using `.env` for managing your mnemonics and other ke
 MNEMONIC="jar deny prosper gasp flush glass core corn alarm treat leg smart"
 INFURA_KEY="<Your Infura Project ID>"
 RINKEBY_MNEMONIC="<Your Rinkeby Mnemonic>"
+GOERLI_MNEMONIC="<Your Goerli Mnemonic>"
```

Comment:
I started adding more explanation here about Nitro, but decided it might be overkill.  Curious what others think.

Reply:
Yeah no need to add it here I think.

Reply:
Can it be simply documented in the READMEmd? If so, it's probably better done there.

Reply:
@dkillen this is the README.md. Are you saying that we should add additional explanation about Nitro here?

Reply:
Oops. My mistake. I thought I was looking at the config. With respect to explanation of Nitro, I think the only content that should be in the README.md is that which is necessary to use the box. It looks fine to me as is, but if you consider the user would benefit from some further explanation maybe it should be added.

Reply:
Okay, I'll leave the explanation out since Nitro will be the status quo moving forward.